WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -801,13 +836,12 @@ import WordPressComAnalytics
     // MARK: - Sync Methods
 
     func updateLastSyncedForTopic(objectID:NSManagedObjectID) {
-        do {
-            let topic = try managedObjectContext().existingObjectWithID(objectID) as! ReaderAbstractTopic
-            topic.lastSynced = NSDate()
-            ContextManager.sharedInstance().saveContext(managedObjectContext())
-        } catch let error as NSError {
-            DDLogSwift.logError("Failed to update topic last synced date: \(error.localizedDescription)")
+        guard let topic = (try? managedObjectContext().existingObjectWithID(objectID)) as? ReaderAbstractTopic else {
+            DDLogSwift.logError("Failed to retrive an existing topic when updating last sync date.")
```

Comment:
I noticed you've changed quite a number of `do / try / catch` statements to just a `try?`. Previously it looks like we were catching the error and logging the description – is that error information not useful?